src/components/plots/gridPlot.ts

```diff
@@ -48,17 +48,14 @@ export module Plots {
       return [{attrToProjector: this._generateAttrToProjector(), animator: this._getAnimator("cells")}];
     }
 
-    public x(): Plots.AccessorScaleBinding<any, number>;
-    public x(x: number | _Accessor): Grid;
-    public x(x: any | _Accessor, scale: Scale<any, number>): Grid;
-    public x(x?: number | _Accessor | any, scale?: Scale<any, number>): any {
+    public x(x?: number | Accessor<number> | any | Accessor<any>, scale?: Scale<any, number>): any {
```

Comment:
Technically out of scope but let's see if I can make the conversion without any problems